rust/agama-migrate-wicked/src/interface.rs

```diff
@@ -279,15 +281,29 @@ impl From<&Interface> for IpConfig {
         .unwrap();
 
         let mut addresses: Vec<IpInet> = vec![];
-        if val.ipv4_static.is_some() {
-            addresses.push(
-                IpInet::from_str(val.ipv4_static.as_ref().unwrap().address.local.as_str()).unwrap(),
-            );
+        if val.ipv4_static.is_some() && val.ipv4_static.as_ref().unwrap().addresses.is_some() {
+            for address in val
+                .ipv4_static
+                .as_ref()
+                .unwrap()
+                .addresses
+                .as_ref()
+                .unwrap()
+            {
+                addresses.push(IpInet::from_str(address.local.as_str()).unwrap());
+            }
```

Comment:
```suggestion
        if let Some(ipv4) = &val.ipv4_static {
            if let Some(addresses_in) = &ipv4.addresses {
                for addr in addresses_in {
                    addresses.push(IpInet::from_str(addr.local.as_str()).unwrap());
                }
            }
        }
```
WDYT?

Reply:
Yep, seems more readable and also may be better in the future when handling more stuff from the ipv*_static structs